README.rst

```diff
@@ -41,6 +41,26 @@ In addition, recent versions of the WeMo firmware may not accept connections fro
 
 The ``setup_url_for_address`` function will lookup a hostname and provide a suitable ``url`` with an IP address.
 
+If the WeMo device is not on your network, you can also connect to it directly.
+After connecting, if the ``pywemo.discover_devices()`` doesn't work, you can get the IP Address by running an ``arp -a`` and use that in ``pywemo.setup_url_for_address``:
+
+.. code-block:: bash
```

Comment:
I think removing 'bash' here might fix the test. I don't know why the test fails for the bash output but succeeds for Python output.

Reply:
Oh weird, I've gone and removed it.